dockerfiles/dartagnan.dockerfile

```diff
@@ -28,26 +28,22 @@ RUN update-ca-certificates
 
 # Install essential stuff ######################################################
 RUN apt-get update && apt-get install -y \
-    git build-essential cmake maven \
-    lsb-release sudo wget \
-    software-properties-common
-
-# Install SMACK ################################################################
-RUN cd home && \
-    git clone -b develop https://github.com/smackers/smack.git && \
-    cd smack && \
-    git checkout 497740c
-
-ENV DOTNET_CLI_TELEMETRY_OPTOUT=1
-RUN cd home && \
-    cd smack && \
-    sed -i 's/sudo /sudo -E /' bin/build.sh && \
-    env TEST_SMACK=0 INSTALL_Z3=0 INSTALL_CORRAL=0 bin/build.sh
+        git \
+        build-essential \
+        cmake \
+        maven \
+        lsb-release \
+        sudo \
+        wget \
+        software-properties-common \
+        graphviz \
+        openjdk-17-jdk \
+        openjdk-17-jre
 
 # Install Dat3M ################################################################
 RUN cd home && \
     git clone https://github.com/hernanponcedeleon/Dat3M.git && \
-    cd Dat3M && git checkout e59dbcb928030380ef62602879a34e2f62817cd5
+    cd Dat3M && git checkout d0b2777be3f923bfd6ef3ba398fa5499705a82a0
```

Comment:
This is the most up-to-date hash in the development branch, right?

Reply:
Yes I took the top commit of development branch. Is that what we should do?

Reply:
yes